constellations/gemini-admin/src/account/accountInfo.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+require("./accountInfo.sass");
+import React from "react";
+import { Link } from "react-router-dom";
+
+export class AccountInfo extends React.Component {
+    formatCurrency = (amount) => {
+        // if (amount < 0) {
+        //     return "-$" + (-amount).toString();
+        // } else {
+        //     return "$" + amount;
+        // }
+        return new Intl.NumberFormat("en-US", {
+            style: "currency",
+            currency: "USD",
+        }).format(amount);
+    };
+
+    render = () => {
+        const id = this.props.id;
+        const email = this.props.email;
+        const users = this.props.users;
+        const transactions = this.props.transactions;
+        const userAddLink = "/users/" + this.props.id + "/add";
+
+        const userRows = users.map((user, index) => {
+            var name = user.firstName + " ";
+            if (user.middleName) {
+                name += user.middleName + " " + user.lastName;
+            } else {
+                name += user.lastName;
+            }
```

Comment:
Could make this more Javascript-y?

name = (user.firstName + " ") + (user.middleName ? (user.middleName + " " + user.lastName) : user.lastName);
OR
name = user.firstName + " ";
name += (user.middleName ? user.middleName + " " : "");
name += user.lastName